notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -18,6 +18,53 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
 Collection: Science fiction films of the 2000s
 
 
+movie: Supernova (2000)
+=======================
+
+:: Title
+Supernova
+
+:: Description
+Supernova is a 2000 science fiction horror film written by David C. Wilson,
+William Malone and Daniel Chuba and directed by Walter Hill, credited as
+"Thomas Lee."
+
+Synopsis: A search and rescue medical ship crew in deep space discovers a
+potentially dangerous alien artifact form after responding to a distress
+signal.
+
+:: Date
+2000-01-14
+
+:: References
+https://en.wikipedia.org/wiki/Supernova_(2000_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+sun going supernova [The story culminated with the revelation that the nine-dimensional matter inside the alien artifact would make the blue giant star go supernova, and moreover that the resulting explosion would reach Earth within 51 years. This, the ship computer hypothesized, would either destroy life on Earth or "enable humankind to achieve a new level of existence".]
+speculative spaceship [The film is largely set aboard the 22nd century, faster-than-light traveling medical rescue vessel Nightengale 229. Another novel aspect of the ship is that it used rotation to generate gravity.]
+romantic love [Nick and Kaela fell in love in the end. Danika and Yerzy were in love and longed to have children togehter.]
+old flames [Kaela was astonished to discover that the man who'd sent the distress call was her former lover, Karl. Karl's attempts to rekindle their relationship was central to the plot.]
+unrequited love [One storyline follows Karl in his unsuccessful attempts to romantically reconcile with his former lover Kaela.]
+
+:: Minor Themes
+violence in the media [It was mentioned that violent cartoons, such as Tom and Jerry, had been banned at some point in the 21st century.]
+the medical professions [The ship physician Dr. Kaela Evers was depicted treating crew members.]
+faster than light travel [The medical rescue vessel Nightengale 229 made a "dimension-jump" of 3,000 light-years to reach the ice moon Titan 37 in response to a distress call.]
+space colonization [The film is set in the 22nd century when humanity has colonized some small portion of the galaxy.]
+tabletop game [Benjamin played chess with the ship computer.]
+human overpopulation [Danika's lover assured her that they'd get approved to have a baby, suggesting that some population control measures were in place.]
```

Comment:
Interesting. I can picture the situation and imagine I would agree with your assessment. But how do we motivate that it was related to overpopulation, and not just some other futuristic social idea? Perhaps they were a discriminated against group? Or gov had decided to impose restrictions for some other reason than what we would dub "overpopulation". Was there any other indication of overpopulation?

Reply:
I grappled with this one. What was I think clear is that governmental approval was required in order to have a child. But the underlying reason for this policy was never explored. I'm fine to just delete the **human overpopulation** theme here. Another option is to introduce a theme about governments controlling peoples reproduction.

Thoughts?